src/Console/FlushCommand.php

```diff
@@ -34,16 +34,8 @@ public function handle(Dispatcher $events)
 
```

Comment:
@driesvints This class no longer needs the line `use Laravel\Scout\Events\ModelsFlushed;`.

Reply:
Done!